src/libnvme-mi.map

```diff
@@ -0,0 +1,11 @@
+{
+	global:
+		nvme_mi_init_ctrl;
+		nvme_mi_close;
+		nvme_mi_mi_read_mi_data_subsys;
+		nvme_mi_mi_read_mi_data_port;
```

Comment:
The naming scheme might be consistent. As German speaker I have to lough just heargin mimi :)
I leave these details to Keith and Hannes though :)

Reply:
Yeah, this is for a structured namespace, really. The base "prefix" here is `nvme_mi`, the MI commands are given an additional `_mi`, the admin functions use `_admin`.

If there's a better naming scheme, I'm ok to work with that.

Reply:
Let's keep it consistent and I can giggle everytime I read it :)

Reply:
ok, I'm happy to provide both consistency and amusement. :laughing: